src/module.rs

```diff
@@ -152,9 +118,6 @@ impl<'a> Module<'a> {
             .map(Segment::ansi_string)
             .collect::<Vec<ANSIString>>();
 
-        ansi_strings.insert(0, self.prefix.ansi_string());
-        ansi_strings.push(self.suffix.ansi_string());
-
```

Comment:
Now that we are not modifying `ansi_strings` anymore it does not have to mut anymore. But the match statement has to be changed to return it's value. So something like this:
```rust
match shell {
    Shell::Bash => ansi_strings_modified(ansi_strings, shell),
    Shell::Zsh => ansi_strings_modified(ansi_strings, shell),
    _ => ansi_strings,
}
```

Reply:
If you'd like, you can directly push that to my branch as this is a minor change. Or I'll do that this Saturday.

Reply:
I directly pushed it so that we can get things moving a bit quicker.